pymc3/sampling.py

```diff
@@ -280,6 +283,37 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
     return sample_func(**sample_args)[discard:]
 
 
+def _check_start_shape(model, start):
+    e = ''
+    for var in model.vars:
+        if np.shape(start):
```

Comment:
Shouldn't this be outside of the loop over the vars?
The check if `np.shape()` is a bit strange. How about using `isinstance(start, (collections.abc.Sequence, np.ndarray))` instead? I didn't know that `np.shape` returns an empty tuple before I checked in the interpreter, and I think most people wouldn't.